src/components/Cell.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+type CellProps = {
+    classes: {
+        cell: string
+    },
+    letter: null
```

Comment:
Let's rename this to value or symbol, letter it's too generic.

Reply:
Why are you typing it as null, it should be a string, but also it should only accept two values: X or O.
```suggestion
    letter: 'X' | 'O'
```

Reply:
You right, I am defining the typing not the value!